tests/x11/chromium.pm

```diff
@@ -14,6 +14,16 @@ use warnings;
 use testapi;
 use utils;
 
+# Prevent lost characters due to temporary unresponsiveness of chromium address bar.
+# It looks like chromium becomes unresponsive after typing about 3
+# characters, likely due to some auto-completion feature.
+# See https://progress.opensuse.org/issues/109737 for details
+sub enter_address {
+    my $string = shift;
```

Comment:
I know we're not having a `$self` but  it just looks nicer this way.

```suggestion
    my $string = @_;
```

Reply:
It can look even nicer with proper signatures. Adapted the file accordingly